docs/src/tutorials/standalone/Soil/phase_change_analytic.jl

```diff
@@ -211,7 +211,7 @@ z = parent(soil.domain.fields.z)[:];
 lines!(ax1, sol_T, z, label = "Model", color = :blue, linewidth = 3)
 
 # # Analytic Solution of Neumann
-# All details here are taken from [DallAmico2011](@cite) (see also [CarslawJaeger](@cite)), and the reader is referred to that
```

Comment:
Link to https://clima.github.io/ClimaLand.jl/previews/PR1413/generated/standalone/Soil/phase_change_analytic/@ref%20Soil/soil_energy_hydrology.md does not work